src/payment/is-hosted-intrument-like.spec.ts

```diff
@@ -0,0 +1,54 @@
+import isHostedInstrumentLike from './is-hosted-intrument-like';
+import { getHostedInstrument, getPartialHostedInstrument } from './payments.mock';
+
+describe('isHostedInstrumentLike', () => {
+    it('returns true if the object has both matching properties as undefined', () => {
+        const paymentData = getHostedInstrument(undefined, undefined);
+        expect(paymentData && isHostedInstrumentLike(paymentData)).toBeTruthy();
```

Comment:
It's best to only have the unit under test executed with an `expect` block.

`expect(isHostedInstrumentLike(paymentData)).toBe(true)`

Your existence check on `paymentData` is redundant anyhow as Typescript guarantees a truthy output from your `getHostedInstrument` function.

It's also best to use as explicit an expectation as possible. I.e. `toBe(true)` instead of `toBeTruthy()` 

Lastly, the `getHostedInstrument` is an over optimisation. With tests it's better to be explicit and avoid indirection, rather than worrying about DRY.

I.e.
```
      expect(
        isHostedInstrumentLike({
          shouldSaveInstrument: undefined,
          shouldSetAsDefaultInstrument: undefined,
        })
      ).toBe(true);
```